atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -41,7 +41,13 @@ def run(self):
         """
         pull base image
         """
-        base_image = self.workflow.builder.base_image
+        build_json = get_build_json()
```

Comment:
It would be really great if this was encapsulated in a small helper method in this plugin: `base_image = self.resolve_base_image()`. This method is already large.

Reply:
Fixed.